spec/mailers/user_mailer_spec.rb

```diff
@@ -60,7 +60,7 @@
     context 'with blank notes' do
       let!(:grant_submission) {
         FactoryGirl.create(:grant_submission, artist: artist, grant: grant,
-          requested_funding_dollars: 5000, granted_funding_dollars: 800,
+          funding_requests_csv: 5000, granted_funding_dollars: 800,
```

Comment:
funding_requests_csv is similar, but not the same as, funding_levels_csv.  It's a comma-separated list of individual integers (see grant_submission.rb:116).  So these will all have to change to pass the correct type of value.